minetestmapper

```diff
@@ -0,0 +1 @@
+../minetestmapper
```

Comment:
This is a separate thing. Configurability is already present via the MAPPERPATH env var.